README.md

```diff
@@ -19,8 +19,12 @@ The project is being organized on [Trello](https://trello.com/b/jikk5lqR/sigwel)
 - Hosting (i.e. AWS, Heroku)
 - Client side scaffolding
 
+## Before running the Server
```

Comment:
Thanks for adding this.  I had a prerequisites section at one point but I guess I lost it somehow.